linera-service/src/linera.rs

```diff
@@ -1933,6 +1937,16 @@ async fn main() -> Result<(), anyhow::Error> {
                 eprintln!("\nDone.");
                 Ok(())
             }
+
+            NetCommand::Helper => {
+                info!("You may append the following script to your `~/.bash_profile` or `source` it when needed.");
+                info!(
+                    "This will install a function `linera_spawn_and_read_wallet_variables` to facilitate \
+                       testing with a local Linera network"
```

Comment:
(Lines too long.)

Reply:
For a shell?

Reply:
No, I mean rustfmt would normally make every line ≤ 100 characters, but it doesn't want to touch string literals.